src/classes/filePicker.py

```diff
@@ -0,0 +1,115 @@
+"""
+ @file filePicker.py
```

Comment:
I'm confused by this filePicker.py approach? Is there a reason we can't use a general folder picker: `QFileDialog.getOpenFileUrls`, or something similar? Do we need to create our very own `QFileDialog`?

Reply:
I'm worried that instead of a 1 or 2 line solution, we've introduced a bunch of unneeded complexity to maintain

Reply:
It started with me thinking there was an existing widget for typing in file paths, and suggesting sub-directories.

When I didn't find one, I figured we could make one to plug into other widgets, like properties view, or exporter. A quick way to find files without popping up a dialog.

Reply:
I see that it _is_ scope creep for this PR. So if you like, I can:

1) Move this widget to a new branch for sometime down the line.
2) Move this code (minus the auto-complete) into exportClips.py